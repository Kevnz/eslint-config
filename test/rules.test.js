const eslintConfig = require('../eslint-config.json')



describe('The rules', () => {
  it('should not change', () => {
    expect(eslintConfig.rules).toMatchSnapshot()
  })
})
