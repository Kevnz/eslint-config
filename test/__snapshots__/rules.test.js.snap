// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The rules should not change 1`] = `
Object {
  "arrow-body-style": Array [
    0,
    "as-needed",
  ],
  "comma-dangle": Array [
    "error",
    Object {
      "arrays": "always-multiline",
      "exports": "always-multiline",
      "functions": "never",
      "imports": "always-multiline",
      "objects": "always-multiline",
    },
  ],
  "import/first": Array [
    2,
    "absolute-first",
  ],
  "import/no-unresolved": 2,
  "import/order": Array [
    "error",
    Object {
      "groups": Array [
        "builtin",
        "external",
        "internal",
        "parent",
        "sibling",
        "index",
      ],
    },
  ],
  "no-console": Array [
    1,
    Object {
      "allow": Array [
        "info",
        "warn",
        "error",
      ],
    },
  ],
  "no-only-tests/no-only-tests": 2,
  "no-param-reassign": Array [
    0,
  ],
  "prettier/prettier": Array [
    "error",
    Object {
      "arrowParens": "avoid",
      "bracketSpacing": true,
      "jsxBracketSameLine": false,
      "printWidth": 80,
      "semi": false,
      "singleQuote": true,
      "tabWidth": 2,
      "trailingComma": "es5",
      "useTabs": false,
    },
  ],
  "react/display-name": Array [
    0,
  ],
  "react/prop-types": Array [
    0,
  ],
  "spellcheck/spell-checker": Array [
    1,
    Object {
      "comments": true,
      "identifiers": true,
      "lang": "en_US",
      "minLength": 3,
      "skipIfMatch": Array [
        "http://[^s]*",
        "^[-\\\\w]+/[-\\\\w\\\\.]+$",
      ],
      "skipWords": Array [
        "aff",
        "api",
        "apollo",
        "archiver",
        "args",
        "async",
        "auth",
        "aws",
        "blipp",
        "cors",
        "defs",
        "del",
        "dev",
        "dict",
        "dom",
        "ejs",
        "err",
        "eslint",
        "filename",
        "func",
        "github",
        "gql",
        "graphql",
        "hapi",
        "html",
        "http",
        "hunspell",
        "hunspellchecker",
        "iam",
        "inmemory",
        "joi",
        "jss",
        "jsx",
        "kev",
        "knex",
        "lodash",
        "middleware",
        "mjs",
        "ncc",
        "nodejs",
        "params",
        "pre",
        "readdir",
        "readis",
        "resolvers",
        "rfe",
        "runtime",
        "sdk",
        "serverless",
        "src",
        "stderr",
        "stdin",
        "stdout",
        "uploader",
        "uri",
        "utf",
        "utf8",
        "utf16",
        "util",
        "utils",
        "webpack",
        "xml",
        "xtconf",
        "yml",
        "yaml",
        "zeit",
      ],
      "strings": true,
    },
  ],
  "standard/computed-property-even-spacing": 0,
}
`;
